kore/src/main/scala/org/kframework/unparser/Unparse.scala

```diff
@@ -34,9 +36,13 @@ object Unparse extends {
 object ToKast {
   def apply(k: K): String = {
     val b = StringBuilder.newBuilder
-    unparse(b, false, 0, k)
+    unparse(s => b ++= s, false, 0, k)
```

Comment:
the anonymous function in this context requires looking at what unparse is doing, it could be more readable if it was a named function. 
don't have strong feelings though -- it might be just me getting used to the code base though